modules/core/src/main/java/org/apache/fluo/core/oracle/OracleClient.java

```diff
@@ -73,7 +74,8 @@
   private static final class TimeRequest implements Callable<Stamp> {
     CountDownLatch cdl = new CountDownLatch(1);
     AtomicReference<Stamp> stampRef = new AtomicReference<>();
-    ListenableFutureTask<Stamp> lf = null;
+    // ListenableFutureTask<Stamp> lf = null;
+    CompletableFuture<Stamp> cf = null;
 
     @Override
     public Stamp call() throws Exception {
```

Comment:
Can this call method be removed now?

Reply:
Yes I think you're right. The only reason I see that TimeRequest implemented Callable was so we could create a ListenableFutureTask with it.